content/spot-instances-resiliency/docs/copying-shuffle-data.md

```diff
@@ -1,97 +0,0 @@
-# **Spot instances interruption resiliency by copying shuffle data**
```

Comment:
I feel some of this content is useful. For ex, https://aws.github.io/aws-emr-containers-best-practices/spot-instances-resiliency/docs/copying-shuffle-data/#how-does-it-work. and other sections such as "Observed Behavior" and "Node decommissioning with fallback storage". 

Can we create separate doc focusing on decommissioning feature and merge this content?

Reply:
This has been addressed as discussed, please review